bin/runnerman.js

```diff
@@ -1,63 +1,76 @@
 #!/usr/bin/env node
 
+const { program } = require('commander');
 const { parseJsonFile } = require('../lib/util/parseJsonFile');
 const { getFilesInFolderPerExtension } = require('../lib/util/getFilesInFolderPerExtension');
 const { runnerman } = require('..');
-const { program } = require('commander');
+const { awaitLast } = require('../lib/util/awaitLast');
 const { version } = require('../package.json');
 
 const collect = (value, previous) => previous.concat([value]);
 
 program
-  .version(version)
-  .option('-c, --collection <type>')
-  .option('-e, --environment <type>')
-  .option('-s, --suite <value>', '', collect, [])
-  .option('-i, --iterations <type>')
-  .parse(process.argv);
+    .version(version, '-v, --version')
+    .requiredOption('-c, --collection <type>')
+    .option('-e, --environment <value>')
+    .option('-s, --suite <value>', '', collect, [])
+    .option('-i, --iterations <number>')
+    .option('-p, --parallelize')
+    .parse(process.argv);
+
+(async function main(option) {
+    if (option.collection === '' ||
+      option.environment === '' ||
+      option.suite.length === 0) {
+        option.help();
+    }
 
-(async function main(program) {
-  if (program.collection === '' ||
-      program.environment === '' ||
-      program.suite === undefined ||
-      program.suite.length === 0)
-  {
-    program.help();
-  }
+    const collection = parseJsonFile(option.collection);
 
-    const collection = parseJsonFile(program.collection);
+    const environment = option.environment && parseJsonFile(option.environment);
 
-    const environment = program.environment !== undefined && parseJsonFile(program.environment);
+    const iterations = parseInt(option.iterations, 10) || 1;
+
+    const items = [];
+
+    for (const item of option.suite) {
+        items.push(getFilesInFolderPerExtension(item, 'suite'));
+    }
 
-    const iterations = parseInt(program.iterations) || 1;
+    await Promise.all(items);
 
-  const suites = new Set();
+    const suites = new Set();
 
-  for (const item of program.suite) {
-    var items = await getFilesInFolderPerExtension(item, "suite");
-    items.forEach(x => suites.add(x));
-  };
+    // TODO:  Verify if exist batter approach
+    items.forEach(element => element.forEach(x => suites.add(x)));
 
     const summaries = [];
 
-    for (const suite of suites) {
-        const summary = await runnerman(
-            {
+    const resolveds = [];
```

Comment:
```suggestion
    const revolvedPromises = [];
```
Trocar nos lugares onde é utilizada